spec.bs

```diff
@@ -9262,6 +9299,62 @@ An <dfn>auction data buyer config</dfn> is a [=struct=] with the following [=str
     size of the request to allocate to this buyer.
 </dl>
 
+<h3 id=cumul-timeout-tracker>Cumulative timeout tracker</h3>
+
+A <dfn>cumulative timeout tracker</dfn> is a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="cumulative timeout tracker">
+  : <dfn>remaining</dfn>
+  :: A [=duration=] in milliseconds or null. When non-null, records how much of the timeout
+    has been consumed. When null, states there is no limit. Initially null.
+  : <dfn>limit</dfn>
+  :: A [=duration=] in milliseconds, or null.

```

Comment:
also say that it means no limit when it's null

Reply:
Done.